docs/platforms/native/advanced-usage/backend-tradeoffs/index.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Backend Tradeoffs
+description: "How to choose the right crash backend in the Native SDK."
+sidebar_order: 1000
+---
+The Native SDK lets users decide at compile-time between three crash backends:
+
+* `crashpad`
+* `breakpad`
+* `inproc`
+
+Currently, `crashpad` is the default on all desktop platforms because it
+
+* has an external `handler` process that allows for external snapshots and sending crash reports immediately (instead of on the next successful start of your application)
+* is the primary target for extension compared to upstream, including
+  * client-side stack traces
+  * attachment handling
+  * HTTP proxy support
+  * CMake build scripts
+  * GCC and MinGW support
+  * `FirstChanceHandler` on Windows and extension of its synchronization to support Sentry hooks
+  * cooperation with Epic's Easy Anti Cheat
+* supports more error types on [Linux](/platforms/native/advanced-usage/signal-handling/#signals-of-interest) and Windows (`abort()` and other `fast-fail` crashes, handling of heap corruptions)
+* is more maintained upstream (although most changes affect new platforms like Fuchsia)
+
+### When shouldn't I use the `crashpad` backend?
+
+Sentry decided on `crashpad` as the default on all platforms because there are a lot of upsides. However, there are use cases where `crashpad` cannot be used or makes distribution or deployment much harder. We provide other backends for situations when
+
+* you cannot package or deploy an additional executable (the `crashpad_handler`)
```

Comment:
is it useful to add 'or spawn'? Given environments where there's a sandbox and we can't create child processes? (Xbox and UWP I believe?)

```suggestion
* you cannot package, deploy or spawn an additional executable (the `crashpad_handler`)
```

I see this might be covered in the next two lines though

Reply:
I wanted to differentiate cases in which even packaging separate executables was a problem from cases in which a runtime inhibition exists.